clr/commands.py

```diff
@@ -273,7 +273,10 @@ def cmd_help(self, query=None, query2=None):
         self.print_help_for_command(namespace_key, command_name)
 
     def print_help_for_command(self, namespace_key, command_name, prefix=''):
-        width = os.get_terminal_size().columns
+        try:
+            width = os.get_terminal_size().columns
```

Comment:
I realize this isn't your code, but `os.get_terminal_size.__doc__` reads:

> `shutil.get_terminal_size` is the high-level function which should
> normally be used, `os.get_terminal_size` is the low-level implementation.

Should we swap over?